### PR TITLE
Add Troubleshooting page

### PR DESCRIPTION
**Pull request by Pacyfik**
Ticket: https://trello.com/c/2I95z3zI/197-dashboard-review-faqs-related-to-dashboard-and-find-a-better-place-for-them-in-the-dashboard

Idea for the page came up in discussions [about this PR](https://github.com/KnapsackPro/knapsack-pro-api/pull/888).

1. Troubleshooting page:
00a-docs
00b-docs

2. `KNAPSACK_PRO_BRANCH` change:
00c-branch

### CODE REVIEW

**Comment by Pacyfik** [comment]
@3v0k4 Hey. I'd ideally like to use a different icon for the new troubleshooting tile on the Welcome page. Would you know how I could go about coming up with one? Do we have some sort of asset storage somewhere?

**Comment by 3v0k4** [comment]
@Pacyfik if you add a ticket with some description, I'll take care of creating one.

**Comment by Pacyfik** [comment]
> @Pacyfik if you add a ticket with some description, I'll take care of creating one. I do that in Figma.

@3v0k4 Thanks! I've created [the ticket](https://trello.com/c/cVPQPd4N/216-create-an-icon-for-the-troubleshooting-tile-in-the-docs) as you asked. I don't think using the existing knapsack icon blocks this PR to be honest. It can be replaced later based on the new ticket.

File: docusaurus/docs/index.mdx [hunk 1]
```diff
@@ -46,10 +46,19 @@ If you have questions, feel free to [contact us](https://knapsackpro.com/contact
 
 Can't see your language/testing framework above? [Let us know](https://docs.google.com/forms/d/e/1FAIpQLSe7Z6k__VczmRMmXykjA5i2MVEA3nEJ90gbiIeCRjecWhPOig/viewform?hl=en).
 
-You can still use Knapsack Pro by writing your integration on top of the JavaScript SDK or using the Knapsack Pro API:
+## Troubleshooting
```
Comment (3v0k4) [inline docusaurus/docs/index.mdx:49]: I'd put this at the bottom.

I believe "Can't see your language/testing framework above?" should be close to "Write your integration".

File: docusaurus/docs/troubleshooting.mdx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+pagination_prev: null
+pagination_next: null
+---
+
+# Troubleshooting
+
+## When using two different CI providers, why are not all test files executed?
+
+If you are utilizing two CI providers simultaneously, for example, CircleCI and TravisCI, and you run a test suite, it is necessary to have a separate API token for tests executed on CircleCI and another separate API token for the test suite executed on TravisCI.
```
Comment (3v0k4) [inline docusaurus/docs/troubleshooting.mdx:10]: Maybe we should add the links for the troubleshootings of each library at the bottom?

Similar to what we have in the welcome page:

Screenshot 2024-02-26 at 16 09 59

**Review by 3v0k4 — COMMENTED** [review]
Left a couple of suggestions. Feel free to apply them or not and merge.

File: docusaurus/docs/troubleshooting.mdx (hunk 2 above)
Comment (Pacyfik) [reply docusaurus/docs/troubleshooting.mdx:10]: Does this look ok?
Troubleshooting___Knapsack_Pro_Docs

Comment (Pacyfik) [reply docusaurus/docs/troubleshooting.mdx:14]: I actually based it on the index page and this part: `Can't see your language/testing framework above?` But libraries will work even better.

**Pull request merged**